src/main/java/teammates/storage/api/EntitiesDb.java

```diff
@@ -288,6 +289,18 @@ protected void putDocument(String indexName, SearchDocument document) {
         }
     }
 
+    protected void putDocuments(String indexName, List<SearchDocument> documents) {
+        List<Document> searchDocuments = new ArrayList<Document>();
+        for (SearchDocument document : documents) {
+            searchDocuments.add(document.build());
+        }
+        try {
+            SearchManager.putDocuments(indexName, searchDocuments);
+        } catch (Exception e) {
+            log.info("Failed to batch put searchable documents in " + indexName + " for " + documents.toString());
+        }
```

Comment:
Is this supposed to fail frequently and without serious consequences? I think not. In that case, should be `severe`.